style.css

```diff
@@ -0,0 +1,63 @@
+body {
+    padding: 10px 15px;
+    display: flex;
+}
+
+div.input {
+    width: calc(100% - 17em);
+}
+
+div.damageTable {
+    position: fixed;
+    right: 0;
+    width: 17em;
+}
+
+.form-control, .input-group-addon {
+    padding: 5px;
+}
+
+input.form-control {
+    width: auto;
+    display: inline-block;
+}
+
+input#armor {
+    width: 5em;
+}
+
+input#charges, input#resistance {
+    width: 2.5em;
+}
+
+.table {
+    width: auto;
+    margin-bottom: auto;
+}
+
+.table > tbody > tr > td {
+    vertical-align: middle;
+    border-top: inherit;
+}
+
+.table td.name > input {
+    width: 10em;
+}
+
+.table td.percent > * {
+    width: 4em;
+    text-align: right;
+}
+
+.table td > input#flatTaken {
+    width: 2.75em;
+}
+
+input[type='number']::-webkit-outer-spin-button,
+input[type='number']::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+}
+
+input[type='number'] {
+    -moz-appearance: textfield;
```

Comment:
yeah the webkit stuff is handled in the line above that, I guess I could condense it to one block though? The spinners are just not super useful in this context and take up a lot of space in the standard bootstrap styles. I really just want the number input restriction.